f/spect/fn_activateSpectator.sqf

```diff
@@ -2,18 +2,22 @@ if (f_param_debugMode == 1) then {
     diag_log "activating spectator";
 };
 
-// 'Cinematic' delay before spectator activates
-sleep 3;
+0 spawn {
```

Comment:
Out of interest, why is this block now scheduled (/spawned)?

Reply:
unclear! I'll doublecheck that it isn't necessary

Reply:
Aha, somewhere along the way and only in my mission test file I had a isCall added to the onPlayerRespawn template in description.ext